cms/djangoapps/contentstore/rest_api/v2/views/home.py

```diff
@@ -90,9 +99,16 @@ def get(self, request: Request):
         """
 
         courses, in_process_course_actions = get_course_context_v2(request)
-        courses_context = {
-            "courses": courses,
-            "in_process_course_actions": in_process_course_actions,
-        }
-        serializer = CourseHomeTabSerializerV2(courses_context)
-        return Response(serializer.data)
+        paginator = PageNumberPagination()
+        courses_page = paginator.paginate_queryset(courses, self.request, view=self)
+        serializer = CourseHomeTabSerializerV2({
+            'courses': courses_page,
+            'in_process_course_actions': in_process_course_actions,
+        })
+        return Response(OrderedDict([
+            ('count', paginator.page.paginator.count),
+            ('num_pages', paginator.page.paginator.num_pages),
```

Comment:
**Note to reviewe**
I'm currently studying which parts of the implementation are affecting performance, based on those results, this or the count might have to go.